packages/office-ui-fabric-react/src/utilities/positioning.test.ts

```diff
@@ -474,16 +474,16 @@ describe('getBoundingRectangle', () => {
 
     it('Gets correct bounds in horizontal fold dual screen scenarios when Points are passed as targets', () => {
       const targetWithinLeftScreen = {
-        x: 150,
-        y: 150,
+        left: 150,
+        top: 150,
       } as Point;
```

Comment:
Nitpicky thing but while you're here can you change this to declare the type properly rather than using a cast?